docs/source/index.rst

```diff
@@ -30,7 +30,6 @@ NumPyro documentation
    svi
    autoguide
    reparam
-   funsor
```

Comment:
nit: I agree that having the docs for `funsor` is not important but it is useful to have some urls so that we can link to when users ask questions. :)

Reply:
I removed it because of the sphinx warning. I don't see the link to funsor docs anywhere in http://num.pyro.ai/en/latest/, did you intend to link it somewhere?

Reply:
Hmm, probably I missed that file during a commit. Let me check again.

Reply:
Oops, I missed that file indeed. Could you help me add the following content to `funsor.rst`?
```
Funsor-based NumPyro
====================

.. automodule:: numpyro.contrib.funsor

Effect handlers
---------------

.. autoclass:: numpyro.contrib.funsor.enum_messenger.enum
    :members:
    :undoc-members:
    :show-inheritance:
    :member-order: bysource

.. autoclass:: numpyro.contrib.funsor.enum_messenger.infer_config
    :members:
    :undoc-members:
    :show-inheritance:
    :member-order: bysource

.. autoclass:: numpyro.contrib.funsor.enum_messenger.markov
    :members:
    :undoc-members:
    :show-inheritance:
    :member-order: bysource

.. autoclass:: numpyro.contrib.funsor.enum_messenger.plate
    :members:
    :undoc-members:
    :show-inheritance:
    :member-order: bysource

.. autoclass:: numpyro.contrib.funsor.enum_messenger.to_data
    :members:
    :undoc-members:
    :show-inheritance:
    :member-order: bysource

.. autoclass:: numpyro.contrib.funsor.enum_messenger.to_funsor
    :members:
    :undoc-members:
    :show-inheritance:
    :member-order: bysource

.. autoclass:: numpyro.contrib.funsor.enum_messenger.trace
    :members:
    :undoc-members:
    :show-inheritance:
    :member-order: bysource

Inference Utilities
-------------------

.. autofunction:: numpyro.contrib.funsor.infer_util.config_enumerate

.. autofunction:: numpyro.contrib.funsor.infer_util.log_density

.. autofunction:: numpyro.contrib.funsor.infer_util.plate_to_enum_plate
```

Reply:
Sounds good, I had some more doc related changes. Will put up a separate PR.

Reply:
Thansk for fixing this issue!